src/Console/Shell.php

```diff
@@ -709,19 +709,35 @@ public function hr($newlines = 0, $width = 63)
      * Displays a formatted error message
      * and exits the application with status code 1
      *
+     * @param string|null $message Error message
+     * @param int $exitCode Exit code
+     * @return int Exit code
+     */
+    public function abort($message, $exitCode = 1) {
+        $this->_io->err('<error>' . $message . '</error>');
+        $this->_stop($exitCode);
```

Comment:
If we're building new things, would an exception be better for the future? The ShellDispatcher could catch exceptions and convert their error codes into exit codes?